hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -3894,6 +3894,18 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.snapshot.db.max.open.files</name>
+    <value>100</value>
+    <tag>OZONE, OM</tag>
+    <description>
+      Max number of open files for each snapshot db present in the snapshot cache.
+      This will limit the total number of files opened by a snapshot db thereby limiting the total number of
```

Comment:
```suggestion
      Max number of open files for each snapshot db present in the snapshot cache.
      Essentially sets `max_open_files` config for RocksDB instances opened for Ozone snapshots.
      This will limit the total number of files opened by a snapshot db thereby limiting the total number of
```

Reply:
done